site/content/docs/manifest/rd-web-service.en.md

```diff
@@ -85,7 +85,13 @@ The architecture type for your service. A [Request-Driven Web Service](../concep
 <a id="http" href="#http" class="field">`http`</a> <span class="type">Map</span>  
 The http section contains parameters related to the managed load balancer.
 
-<span class="parent-field">http.</span><a id="http-healthcheck" href="#http-healthcheck" class="field">`healthcheck`</a> <span class="type">String or Map</span>  
+<span class="parent-field">http.</span><a id="http-private" href="#http-private" class="field">`private`</a> <span class="type">Bool or Map</span>
+Restrict incoming traffic to only your environment. Defaults to false.
+
+<span class="parent-field">http.private</span><a id="http-private-endpoint" href="#http-private-endpoint" class="field">`endpoint`</a> <span class="type">String</span>
+If you would like to utilize a VPC Endpoint to App Runner that you created outside of Copilot, you can import its ID here and it will be used instead of creating one.
```

Comment:
Maybe just 🤔 
```suggestion
An existing VPC Endpoint ID for the App Runner service.
```

Reply:
did something similar, see what you think!